util/util.go

```diff
@@ -303,3 +309,17 @@ func CheckArtifactoryLicense(client *resty.Client, licenseTypesToCheck ...string
 type Identifiable interface {
 	Id() string
 }
+
+func CheckVersion(versionToCheck string, supportedVersion string) (bool, error) {
```

Comment:
Do you send a call to query RT version somewhere?

Reply:
Yes, in `providerConfigure()` in `provider.go`. It's waiting for this to merge first before I open a PR in Artifactory repo.